### PR TITLE
Create a single project example to show key features

### PR DESCRIPTION
**Pull request by tmiddlet2666**

### CODE REVIEW

File: examples/deployment/src/main/yaml/example-cluster-persistence.yaml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+          enabled: true
+          persistentVolumeClaim:
+            accessModes:
+            - ReadWriteOnce
+            resources:
+              requests:
+                storage: 1Gi
+      application:
+        image: deployment-example:2.0.0
+        imagePullPolicy: IfNotPresent
```
Comment (thegridman) [inline examples/deployment/src/main/yaml/example-cluster-persistence.yaml]: `IfNotPresent` is the Kubernetes default for `imagePullPolicy` so it isn't really required

File: examples/deployment/src/main/yaml/example-cluster-persistence.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    - role: proxy
+      replicas: 1
+      ports:
+        - name: proxy
+          port: 20000
+      coherence:
+        cacheConfig: proxy-cache-config.xml
+        storageEnabled: false
+    - role: helidon-app
```
Comment (thegridman) [inline examples/deployment/src/main/yaml/example-cluster-persistence.yaml]: Same comment as for the other file regarding role and port names

File: examples/deployment/src/main/yaml/example-cluster-proxy.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    - name: metrics
+      port: 9612
+  coherence:
+    logLevel: 5
+    image: iad.ocir.io/odx-stateservice/test/coherence:12.2.1.4.0-b74630
+    metrics:
+      enabled: true
+  application:
+    image: deployment-example:2.0.0
+    imagePullPolicy: IfNotPresent
```
Comment (thegridman) [inline examples/deployment/src/main/yaml/example-cluster-proxy.yaml]: `IfNotPresent` is the Kubernetes default for `imagePullPolicy` so it isn't really required

File: examples/deployment/src/main/yaml/example-cluster-proxy.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  # Individual cluster roles
+  #
+  roles:
+    - role: storage
+      replicas: 3
+      coherence:
+        cacheConfig: storage-cache-config.xml
+      application:
+        image: deployment-example:2.0.0
+        imagePullPolicy: IfNotPresent
```
Comment (thegridman) [inline examples/deployment/src/main/yaml/example-cluster-proxy.yaml]: `IfNotPresent` is the Kubernetes default for `imagePullPolicy` so it isn't really required

File: examples/deployment/src/main/yaml/example-cluster.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+    - name: metrics
+      port: 9612
+  coherence:
+    logLevel: 5
+    image: iad.ocir.io/odx-stateservice/test/coherence:12.2.1.4.0-b74630
+    metrics:
+      enabled: true
+  application:
+    image: deployment-example:2.0.0
+    imagePullPolicy: IfNotPresent
```
Comment (thegridman) [inline examples/deployment/src/main/yaml/example-cluster.yaml]: `IfNotPresent` is the Kubernetes default for `imagePullPolicy` so it isn't really required

File: examples/pom.xml [hunk 6]
```diff
@@ -0,0 +1,190 @@
...
+    <maven.jar.plugin.version>3.1.2</maven.jar.plugin.version>
+    <maven.jib.plugin.version>1.1.0</maven.jib.plugin.version>
+    <maven.resource.plugin.version>3.1.0</maven.resource.plugin.version>
+    <maven.surefire.plugin.version>2.21.0</maven.surefire.plugin.version>
+    <mockito.version>2.18.3</mockito.version>
+    
+    <!-- if necessary, override with -D -->
+    <coherence.version>12.2.1-4-0</coherence.version>
+
+    <helidon.version>1.0.0</helidon.version>
```
Comment (thegridman) [inline examples/pom.xml]: You might want to change this to the most recent version `1.3.1`

File: examples/deployment/src/main/yaml/example-cluster-proxy.yaml [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+  imagePullSecrets:
+    - name: coherence-example-secret
+  jvm:
+    heapSize: 512m
+  ports:
+    - name: metrics
+      port: 9612
+  coherence:
+    logLevel: 5
+    image: iad.ocir.io/odx-stateservice/test/coherence:12.2.1.4.0-b74630
```
Comment (thegridman) [inline examples/deployment/src/main/yaml/example-cluster-proxy.yaml]: You should not need this, the default image used is 12.2.1.4 and when 12.2.1.4 gets onto OCR we will default to that.

File: examples/deployment/src/main/yaml/example-cluster-proxy.yaml [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+spec:
+  imagePullSecrets:
+    - name: coherence-example-secret
+  jvm:
+    heapSize: 512m
+  ports:
+    - name: metrics
+      port: 9612
+  coherence:
+    logLevel: 5
```
Comment (thegridman) [inline examples/deployment/src/main/yaml/example-cluster-proxy.yaml]: Coherence default log level is already `5`

File: examples/deployment/src/main/yaml/example-cluster.yaml [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+  imagePullSecrets:
+    - name: coherence-example-secret
+  jvm:
+    heapSize: 512m
+  ports:
+    - name: metrics
+      port: 9612
+  coherence:
+    logLevel: 5
+    image: iad.ocir.io/odx-stateservice/test/coherence:12.2.1.4.0-b74630
```
Comment (thegridman) [inline examples/deployment/src/main/yaml/example-cluster.yaml]: You should not need this, the default image used is 12.2.1.4 and when 12.2.1.4 gets onto OCR we will default to that.

File: examples/deployment/src/main/yaml/example-cluster.yaml [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+spec:
+  imagePullSecrets:
+    - name: coherence-example-secret
+  jvm:
+    heapSize: 512m
+  ports:
+    - name: metrics
+      port: 9612
+  coherence:
+    logLevel: 5
```
Comment (thegridman) [inline examples/deployment/src/main/yaml/example-cluster.yaml]: Coherence default log level is already `5`

File: examples/deployment/src/main/yaml/example-cluster-proxy.yaml [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+# Example 2
+# Adds a separate proxy role
+apiVersion: coherence.oracle.com/v1
+kind: CoherenceCluster
+metadata:
+  name: example-cluster
+spec:
+  imagePullSecrets:
+    - name: coherence-example-secret
+  jvm:
```
Comment (thegridman) [inline examples/deployment/src/main/yaml/example-cluster-proxy.yaml]: Should be:
```
  jvm:
    memory:
      heapSize: 512m
```

File: examples/deployment/src/main/yaml/example-cluster.yaml [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+# Example 1
+# Initial cluster with one role of 'storage`
+apiVersion: coherence.oracle.com/v1
+kind: CoherenceCluster
+metadata:
+  name: example-cluster
+spec:
+  imagePullSecrets:
+    - name: coherence-example-secret
+  jvm:
```
Comment (thegridman) [inline examples/deployment/src/main/yaml/example-cluster.yaml]: Should be:
```
  jvm:
    memory:
      heapSize: 512m
```

File: examples/deployment/README.md [hunk 13]
```diff
@@ -0,0 +1,731 @@
...
+The following scenarios are covered:
+
+1. Installing the Coherence Operator 
+1. Installing a Coherence cluster 
+1. Deploying a Proxy tier
+1. Deploying an storage-disabled application 
+1. Enabling Active Persistence
+
+After the initial install of the Coherence cluster, the following examples 
+build on the previous ones by issuing a `helm apply` to modify
```
Comment (thegridman) [inline examples/deployment/README.md]: Shouldn't that be `kubectl apply`

File: examples/deployment/README.md [hunk 14]
```diff
@@ -0,0 +1,731 @@
...
+the install adding additional roles.
+
+You can use `helm create` for any of the examples to install that one directly.
```
Comment (thegridman) [inline examples/deployment/README.md]: Again, we use `kubectl` to create, update and delete Coherence clusters. Helm is only used to install the operator.

File: examples/deployment/README.md [hunk 15]
```diff
@@ -0,0 +1,731 @@
...
+
+# Table of Contents
+
+* [Prerequisites](#prerequisites)
+  * [Coherence Operator Quick Start](#coherence-operator-quick-start) 
+  * [JDK and Maven Vvrsions](#jdk-and-maven-versions)
```
Comment (thegridman) [inline examples/deployment/README.md]: spelling of Versions

File: examples/deployment/README.md [hunk 16]
```diff
@@ -0,0 +1,731 @@
...
+  * [Example 2 - Adding a Proxy role ](#example-2---adding-a-proxy-role) 
+  * [Example 3 - Adding a User application role](#example-3---adding-a-user-application-role)
+  * [Example 4 - Enabling Persistence](#example-4---enabling-persistence)
+* [Cleaning Up](#cleaning-up)  
+
+# Prerequisites
+
+## Coherence Operator Quick Start
+
+Ensure you have followed all the [Quick Start Guide](https://oracle.github.io/coherence-operator/docs-unstable/2.0.0-1910091023/#/about/03_quickstart) including the
```
Comment (thegridman) [inline examples/deployment/README.md]: Hard coding links to pre-release docs is probably a bad idea as we will delete them

File: examples/deployment/README.md [hunk 17]
```diff
@@ -0,0 +1,731 @@
...
+   mvn install:install-file -Dfile=$COHERENCE_HOME/lib/coherence-metrics.jar \
+       -DpomFile=$COHERENCE_HOME/plugins/maven/com/oracle/coherence/coherence-metrics/12.2.1/coherence-metrics.12.2.1.pom
+   ```
+
+## Create the example namespace
+
+You need to create the namespace for the first time to run any of the examples. Create your target namespace:
+
+```bash
+kubectl create namespace coherence-example-ns
```
Comment (thegridman) [inline examples/deployment/README.md]: Do we need `-ns` on the end, it's obviously a namespace

File: examples/deployment/README.md [hunk 18]
```diff
@@ -0,0 +1,731 @@
...
+* Replace `$DOCKER_USERNAME` with your username for that repository.
+* Replace `$DOCKER_PASSWORD` with your password for that repository.
+* Replace `$DOCKER_EMAIL` with your email (or even a fake email).
+
+See the [Kubernetes documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/)
+on pull secrets for more details.
+
+## Clone the GitHub repository
+
+The examples exist in the `2.0` branch of the Coherence Operator GitHub repository - https://github.com/oracle/coherence-operator.
```
Comment (thegridman) [inline examples/deployment/README.md]: You will need to change this when we merge `2.0` onto `master` as once we release we will blow away all the old 1.0 code and 2.0 will be the only source.

File: examples/deployment/README.md [hunk 19]
```diff
@@ -0,0 +1,731 @@
...
+## Install the Coherence Operator
+
+For this example we are going to install the Coherence Operator with metrics enables as well as 
+EFK (Elasticsearch, FluentD and Kiabana) configured for log capture and analysis in Kibana.
+
+If you are short on resources, then you can set `installEFK=false` and `prometheusoperator.enabled=false`.
+
+> Note: If this is the first time you have installed with Prometheus enabled, you should set `createCustomResource=true`.   
+
+> Note: The following uses the `coherence-unstable` helm repository as well as sets a specific pre-release version.
```
Comment (thegridman) [inline examples/deployment/README.md]: You shouldn't be using the unstable repository in the docs, once it is release they should be using 2.0.0 from the release repo

File: examples/deployment/README.md [hunk 20]
```diff
@@ -0,0 +1,731 @@
...
+> If you are using the official 2.0 release, then you can use the `coherence` repository and remove the version to use the latest.
+
+```bash
+helm install coherence-unstable/coherence-operator --version 2.0.0-1910071023 \
```
Comment (thegridman) [inline examples/deployment/README.md]: Should be coming from the stable repo and use 2.0.0 version (or don't specify a version and Helm will fetch the latest version)

File: examples/deployment/README.md [hunk 21]
```diff
@@ -0,0 +1,731 @@
...
+1.  Install the Coherence cluster `storage` role
+
+    ```bash   
+    kubectl -n coherence-example-ns create -f src/main/yaml/example-cluster.yaml 
+    ```       
+
+    List the created `coherenceclusters` and `coherenceroles` using:
+
+    ```bash
+    kubectl get coherenceclusters -n coherence-example-ns
```
Comment (thegridman) [inline examples/deployment/README.md]: You can actually do `kubectl -n coherence-example-ns get coherence` and that will display all `CoherenceCluster` _and_ `CoherenceRole` resources

File: examples/deployment/src/main/yaml/example-cluster-persistence.yaml [hunk 22]
```diff
@@ -0,0 +1,64 @@
...
+spec:
+  imagePullSecrets:
+    - name: coherence-example-secret
+  jvm:
+    heapSize: 512m
+  ports:
+    - name: metrics
+      port: 9612
+  coherence:
+    logLevel: 5
```
[the remark this answers is not in the document]
Comment (tmiddlet2666) [reply examples/deployment/src/main/yaml/example-cluster-persistence.yaml]: fixed

File: examples/deployment/README.md (hunk 15 above)
Comment (tmiddlet2666) [reply examples/deployment/README.md]: fixed

File: examples/deployment/README.md (hunk 19 above)
Comment (tmiddlet2666) [reply examples/deployment/README.md]: yes, understand, was just getting things to work.

File: examples/deployment/src/main/yaml/example-cluster-app.yaml [hunk 23]
```diff
@@ -0,0 +1,56 @@
...
+# Adds a separate application node running helidon app
+apiVersion: coherence.oracle.com/v1
+kind: CoherenceCluster
+metadata:
+  name: example-cluster
+spec:
+  imagePullSecrets:
+    - name: coherence-example-secret
+  jvm:
+    heapSize: 512m
```
[the remark this answers is not in the document]
Comment (tmiddlet2666) [reply examples/deployment/src/main/yaml/example-cluster-app.yaml]: @thegridman The above does not set the memory to 512 It defaults to 2.0GB for my system.
When i use the following, it set the memory.

>   jvm:
>     heapSize: 512m

If you are saying it should be jvm.memory.heapSize, then this is not working.

File: examples/deployment/src/main/yaml/example-cluster-app.yaml [hunk 24]
```diff
@@ -0,0 +1,56 @@
...
+      ports:
+        - name: proxy
+          port: 20000
+      coherence:
+        cacheConfig: proxy-cache-config.xml
+        storageEnabled: false
+    - role: helidon-app
+      replicas: 1
+      ports:
+        - name: helidon
```
[the remark this answers is not in the document]
Comment (tmiddlet2666) [reply examples/deployment/src/main/yaml/example-cluster-app.yaml]: ok sounds good.

File: examples/deployment/README.md (hunk 17 above)
Comment (tmiddlet2666) [reply examples/deployment/README.md]: have fixed this

File: examples/deployment/src/main/yaml/example-cluster-persistence.yaml [hunk 25]
```diff
@@ -0,0 +1,64 @@
...
+    image: iad.ocir.io/odx-stateservice/test/coherence:12.2.1.4.0-b74630
+    metrics:
+      enabled: true
+  application:
+    image: deployment-example:2.0.0
+    imagePullPolicy: IfNotPresent
```
[the remark this answers is not in the document]
Comment (tmiddlet2666) [reply examples/deployment/src/main/yaml/example-cluster-persistence.yaml]: fixed

File: examples/deployment/src/main/yaml/example-cluster-persistence.yaml (hunk 1 above)
Comment (tmiddlet2666) [reply examples/deployment/src/main/yaml/example-cluster-persistence.yaml]: fixed

File: examples/deployment/src/main/yaml/example-cluster-proxy.yaml (hunk 4 above)
Comment (tmiddlet2666) [reply examples/deployment/src/main/yaml/example-cluster-proxy.yaml]: fixed

File: examples/deployment/src/main/yaml/example-cluster.yaml (hunk 5 above)
Comment (tmiddlet2666) [reply examples/deployment/src/main/yaml/example-cluster.yaml]: fixed

File: examples/deployment/src/main/yaml/example-cluster-proxy.yaml (hunk 7 above)
Comment (tmiddlet2666) [reply examples/deployment/src/main/yaml/example-cluster-proxy.yaml]: Will update as mentioned.

File: examples/deployment/src/main/yaml/example-cluster-proxy.yaml (hunk 8 above)
Comment (tmiddlet2666) [reply examples/deployment/src/main/yaml/example-cluster-proxy.yaml]: fixed

File: examples/deployment/src/main/yaml/example-cluster.yaml (hunk 9 above)
Comment (tmiddlet2666) [reply examples/deployment/src/main/yaml/example-cluster.yaml]: fixed

File: examples/deployment/src/main/yaml/example-cluster.yaml (hunk 10 above)
Comment (tmiddlet2666) [reply examples/deployment/src/main/yaml/example-cluster.yaml]: fixed

File: examples/deployment/README.md (hunk 18 above)
Comment (tmiddlet2666) [reply examples/deployment/README.md]: ok understand.

File: examples/deployment/README.md (hunk 20 above)
Comment (tmiddlet2666) [reply examples/deployment/README.md]: understand

File: examples/deployment/README.md (hunk 21 above)
Comment (tmiddlet2666) [reply examples/deployment/README.md]: ok cool.

File: examples/deployment/README.md (hunk 16 above)
Comment (tmiddlet2666) [reply examples/deployment/README.md]: ok thats fine. What will the final link be?

File: examples/deployment/src/main/yaml/example-cluster-app.yaml [hunk 26]
```diff
@@ -0,0 +1,56 @@
...
+  # Individual cluster roles
+  #
+  roles:
+    - role: storage
+      replicas: 3
+      coherence:
+        cacheConfig: storage-cache-config.xml
+      application:
+        image: deployment-example:2.0.0
+        imagePullPolicy: IfNotPresent
```
[the remark this answers is not in the document]
Comment (tmiddlet2666) [reply examples/deployment/src/main/yaml/example-cluster-app.yaml]: fixed

File: examples/deployment/src/main/yaml/example-cluster-app.yaml (hunk 23 above)
[the remark this answers is not in the document]
Comment (tmiddlet2666) [reply examples/deployment/src/main/yaml/example-cluster-app.yaml]: ok this is fixed in alpha-2

File: examples/deployment/src/main/yaml/example-cluster-persistence.yaml [hunk 27]
```diff
@@ -0,0 +1,64 @@
...
+# Example 4
+# Enables active Persistence
+apiVersion: coherence.oracle.com/v1
+kind: CoherenceCluster
+metadata:
+  name: example-cluster
+spec:
+  imagePullSecrets:
+    - name: coherence-example-secret
+  jvm:
```
[the remark this answers is not in the document]
Comment (tmiddlet2666) [reply examples/deployment/src/main/yaml/example-cluster-persistence.yaml]: fixed

File: examples/deployment/src/main/yaml/example-cluster-persistence.yaml (hunk 2 above)
Comment (tmiddlet2666) [reply examples/deployment/src/main/yaml/example-cluster-persistence.yaml]: fixed

File: examples/pom.xml (hunk 6 above)
Comment (tmiddlet2666) [reply examples/pom.xml]: fixed

File: examples/deployment/src/main/yaml/example-cluster-proxy.yaml (hunk 11 above)
Comment (tmiddlet2666) [reply examples/deployment/src/main/yaml/example-cluster-proxy.yaml]: fixed

File: examples/deployment/src/main/yaml/example-cluster.yaml (hunk 12 above)
Comment (tmiddlet2666) [reply examples/deployment/src/main/yaml/example-cluster.yaml]: fixed

File: examples/deployment/src/assembly/image-assembly.xml [hunk 28]
```diff
@@ -0,0 +1,54 @@
...
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+  <id>coherence-assembly</id>
+  <formats>
+    <format>dir</format>
+  </formats>
+
+  <includeBaseDirectory>false</includeBaseDirectory>
+
```
Comment (thegridman) [inline examples/deployment/src/assembly/image-assembly.xml]: Related to the comment on removing the dependency plugin.
Add the following here:
```
  <moduleSets>
    <moduleSet>
      <useAllReactorProjects>true</useAllReactorProjects>
      <includes>
        <include>com.oracle.coherence.kubernetes:deployment-example</include>
      </includes>
      <binaries>
        <outputDirectory>lib</outputDirectory>
        <unpack>false</unpack>
        <outputFileNameMapping>${module.artifactId}.${module.extension}</outputFileNameMapping>
        <dependencySets>
          <dependencySet>
            <outputFileNameMapping>${artifact.artifactId}.${artifact.extension}</outputFileNameMapping>
          </dependencySet>
        </dependencySets>
      </binaries>
    </moduleSet>
  </moduleSets>
```

File: examples/deployment/src/assembly/image-assembly.xml [hunk 29]
```diff
@@ -0,0 +1,54 @@
...
+    </fileSet>
+    <fileSet>
+      <directory>${project.build.directory}/classes/conf</directory>
+      <outputDirectory>files/conf</outputDirectory>
+      <includes>
+        <include>*.xml</include>
+        <include>*.properties</include>
+      </includes>
+    </fileSet>
+    <fileSet>
```
Comment (thegridman) [inline examples/deployment/src/assembly/image-assembly.xml]: Related to the comment on removing the dependency plugin, you can remove this `<fileSet>` section as the changes to the `image-assembly.xml` will add the module's `.jar`

File: examples/deployment/src/assembly/image-assembly.xml [hunk 30]
```diff
@@ -0,0 +1,54 @@
...
+      <directory>${project.build.directory}</directory>
+      <outputDirectory>files/lib</outputDirectory>
+      <includes>
+        <include>*.jar</include>
+      </includes>
+    </fileSet>
+    <fileSet>
```
Comment (thegridman) [inline examples/deployment/src/assembly/image-assembly.xml]: Related to the comment on removing the dependency plugin, you can remove this `<fileSet>` section as the changes to the `image-assembly.xml` will add the dependencies

File: examples/pom.xml [hunk 31]
```diff
@@ -0,0 +1,190 @@
+
```
Comment (thegridman) [inline examples/pom.xml]: You can add the checkstyle plugin to this pom file exactly the same as it has been done in `java/pom.xml` then you can run `mvn package -DskipTests -P checkstyle` to look for errors. 
I expect there will be a few errors as the rules were taken from Helidon (modified slightly by Aleks) and they are very strict.
There is a code style xml file that you can import into IntelliJ `etc/intellij-codestyle.xml` but it does not pick up everything (no @author tags allowed, no trailing spaces on lines, blank lines in imports, packages need package-info.java etc...)

File: examples/deployment/src/main/yaml/example-cluster-app.yaml [hunk 32]
```diff
@@ -0,0 +1,57 @@
...
+  imagePullSecrets:
+    - name: coherence-example-secret
+  jvm:
+    memory:
+      heapSize: 512m
+  ports:
+    - name: metrics
+      port: 9612
+  coherence:
+    image: iad.ocir.io/odx-stateservice/test/coherence:12.2.1.4.0-b74630
```
[the remark this answers is not in the document]
Comment (tmiddlet2666) [reply examples/deployment/src/main/yaml/example-cluster-app.yaml]: ok fixed

File: examples/deployment/pom.xml [hunk 33]
```diff
@@ -0,0 +1,188 @@
...
+  <properties>
+    <maven.compiler.source>8</maven.compiler.source>
+    <maven.compiler.target>8</maven.compiler.target>
+  </properties>
+
+  <dependencies>
+    <!-- module dependencies -->
+    <dependency>
+      <groupId>com.oracle.coherence</groupId>
+      <artifactId>coherence</artifactId>
```
[the remark this answers is not in the document]
Comment (tmiddlet2666) [reply examples/deployment/pom.xml]: fixed

File: examples/deployment/pom.xml [hunk 34]
```diff
@@ -0,0 +1,188 @@
...
+              <appendAssemblyId>false</appendAssemblyId>
+              <descriptors>
+                <descriptor>src/assembly/image-assembly.xml</descriptor>
+              </descriptors>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      
+      <!-- run com.oracle.coherence.examples.SampleClient via mvn exec:java -->
```
[the remark this answers is not in the document]
Comment (tmiddlet2666) [reply examples/deployment/pom.xml]: fixed comment

File: examples/deployment/pom.xml [hunk 35]
```diff
@@ -0,0 +1,188 @@
...
+
+  </dependencies>
+
+  <build>
+    <!-- Use the assembly plugin to pull together the files to use to build the
+         docker image under the target/docker folder -->
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
```
[the remark this answers is not in the document]
Comment (tmiddlet2666) [reply examples/deployment/pom.xml]: Fixed

File: examples/deployment/src/assembly/image-assembly.xml (hunk 28 above)
Comment (tmiddlet2666) [reply examples/deployment/src/assembly/image-assembly.xml]: fixed

File: examples/deployment/src/assembly/image-assembly.xml (hunk 29 above)
Comment (tmiddlet2666) [reply examples/deployment/src/assembly/image-assembly.xml]: fixed

File: examples/deployment/src/assembly/image-assembly.xml (hunk 28 above)
Comment (tmiddlet2666) [reply examples/deployment/src/assembly/image-assembly.xml]: fixed

File: examples/deployment/src/assembly/image-assembly.xml (hunk 30 above)
Comment (tmiddlet2666) [reply examples/deployment/src/assembly/image-assembly.xml]: fixed

File: examples/pom.xml (hunk 31 above)
Comment (tmiddlet2666) [reply examples/pom.xml]: fixed

Comment (tmiddlet2666) [reply examples/pom.xml]: fixed

**Pull request merged**